packages/apps/shopify-app-react-router/src/server/.eslintrc.js

```diff
@@ -1,16 +1 @@
-module.exports = {
-  rules: {
-    'no-restricted-imports': [
-      'error',
-      {
-        patterns: [
-          {
-            group: ['../react/*'],
-            message:
-              'Importing from src/react in src/server is not allowed as they must be kept separate, because it breaks Remix bundling.',
-          },
-        ],
-      },
-    ],
-  },
-};
+module.exports = {};
```

Comment:
I believe we could delete this file. We will just be falling back to the parent eslint config.